src/component/metric/index.tsx

```diff
@@ -35,8 +39,42 @@ function Metric(props: MetricProps) {
   // 4. Props to set on the root node.
   //
   // By default, we are just piping all MetricProps here, but feel free
-  // to do whatever works for you.
-  return <PlasmicMetric {...props} />;
+  // to do whatever works for you. 
+  return <PlasmicMetric
+    name={'Customer Acquisition Cost'}
+    graphContainer={{
+      render: () => <LineChart dataKey={'cac'} data={[
+        {
+          "date": "January 1, 2019",
+          "cac": 50,
+        },
+        {
+          "date": "February 2, 2019",
+          "cac": 55,
+        },
+        {
+          "date": "March 3, 2019",
+          "cac": 40,
+        },
+        {
+          "date": "April 1, 2019",
+          "cac": 35,
+        },
+        {
+          "date": "May 10, 2019",
+          "cac": 39,
+        },
+        {
+          "date": "June 1, 2019",
+          "cac": 40,
+        },
+        {
+          "date": "July 1, 2019",
+          "cac": 50,
+        }
+      ]} />,
+    }} 
+  />;
 }
 
-export default Metric;
+export default Metric;
```

Comment:
As mentioned in another comment, let's have a newline at the end of each file. It might be that you need to configure this in your Atom. Some integrations in editors remove this on save.

Reply:
yup - I think this is a Atom config thing